### PR TITLE
Update Configuration section of README.md

### LINKED ISSUE #44: Add port information to wiki
By default, Opensprinkler uses port 8080 and theirs no documentation that this integration uses port 80. At first, I did not know what was wrong id be great if this documentation was added.

### PR DESCRIPTION
**Pull request by sdwilsh**
This helps explain a few more details that I tripped over, as well as providing a few more pointers to people not as familiar with Home Assistant yet.

Fixes #44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,10 +24,9 @@ Last tested on OS API `2.1.8` and Home Assistant `0.95.0`
       host: <host>
       password: <md5-password>
     ```
-    - Replace `<host>` with the IP address or hostname of your Opensprinkler controller.
-    - Replace `<md5-password>` with the MD5 encrypted version of your password to your Opensprinkler interface.
-
-3. Restart Home Assistant
+    - Replace `<host>` with the IP address or hostname of your Opensprinkler controller.  This should include the port number as well, such as `8080`, the default that OpenSprinkler uses.
```
Comment (vinteo) [inline README.md]: could you quickly add an example 
```
such as `<host>:8080` for port `8080`, the default...
```

**Comment by vinteo** [comment]
fixes #44

File: README.md (hunk 1 above)
Comment (sdwilsh) [reply README.md]: I added a little more detail than that given #44 that you just linked to.

File: README.md [hunk 2]
```diff
@@ -24,10 +24,9 @@ Last tested on OS API `2.1.8` and Home Assistant `0.95.0`
       host: <host>
       password: <md5-password>
     ```
-    - Replace `<host>` with the IP address or hostname of your Opensprinkler controller.
-    - Replace `<md5-password>` with the MD5 encrypted version of your password to your Opensprinkler interface.
-
-3. Restart Home Assistant
+    - Replace `<host>` with the IP address or hostname of your Opensprinkler controller.  This should include the port number as well, such as `8080`, the default that OpenSprinkler uses, unless your install uses port `80`.  For example, `<hostname/id address>:port`.
```
Comment (vinteo) [inline README.md]: small typo, ip address not id address

**Pull request merged**